python/py-markdown-it-py/Portfile

```diff
@@ -2,34 +2,44 @@
 
 PortSystem          1.0
 PortGroup           python 1.0
-PortGroup           github 1.0
-
-github.setup        executablebooks markdown-it-py 2.0.0 v
 
 name                py-markdown-it-py
+version             2.1.0
 revision            0
 categories-append   textproc
-maintainers         {gmail.com:jjstickel @jjstickel} openmaintainer
+maintainers         nomaintainer
 license             MIT
 supported_archs     noarch
 
-description         Markdown parser done right.
+description         Python port of markdown-it
+long_description    {*}${description}. Markdown parser done right.
 
-long_description    This is a Python port of markdown-it, and some of its associated plugins.
+homepage            https://github.com/executablebooks/markdown-it-py
 
-checksums           rmd160  85561d7b0c98daf3250c969118ec22738a1cb064 \
-                    sha256  373e4c670267cba64071389fbfde4ad270f3604b7b49e90bf82c627bff8f7f25 \
-                    size    265351
+checksums           rmd160  6ad74fd9bc0ffc4b61a6c241783e1246179b58d4 \
+                    sha256  cf7e59fed14b5ae17c0006eff14a2d9a00ed5f3a846148153899a0224e2c07da \
+                    size    66227
 
 python.versions     37 38 39 310
 
 if {${name} ne ${subport}} {
-    depends_build-append    port:py${python.version}-setuptools
+    python.pep517   yes
+
+    # diff -NaurdwB ./py-markdown-it-py-orig/pyproject.toml ./py-markdown-it-py-new/pyproject.toml | sed -E -e 's/\.\/py-markdown-it-py-(orig|new)\//.\//' | sed -E -e 's|/opt/local|@@PREFIX@@|g' > ~/Downloads/patch-pyproject_toml.diff
+    patchfiles-append \
+                   patch-pyproject_toml.diff
+    
+    depends_build-append \
+                    port:py${python.version}-flit_core
```

Comment:
instead of this you should specify the backend using `python.pep517_backend flit` then the `python` PG will take care of this (other options are: setuptools [default], poetry, hatch).

Reply:
Thanks done.